--- bin/Ltibutils.pm.orig	2013-06-26 10:40:42.292314961 +0000
+++ bin/Ltibutils.pm	2013-06-26 10:57:26.031112605 +0000
@@ -560,7 +560,8 @@
     binutils         => 'ar --version 2>/dev/null',
     'gcc-c++'        => 'g++ --version 2>/dev/null',
     glibc            => 'ldd --version 2>/dev/null',
-    'glibc-devel'    => sub { -f '/usr/lib/libm.so' || -f '/usr/lib64/libz.so'},
+#    'glibc-devel'    => sub { -f '/usr/lib/libm.so' || -f '/usr/lib65/libz.so'},
+    'glibc-devel'    => sub { -f '/usr/lib/libm.so' || -f '/usr/lib64/libz.so' || -f '/usr/lib/i386-linux-gnu/libm.so'},
     'glibc-headers'  => sub { -f '/usr/include/stdio.h' },
     'libstdc++' => sub {
             return system_nb(<<TXT) == 0;
@@ -582,7 +583,10 @@
                     },
     zlib         => sub { my @f = (glob('/usr/lib/libz.so*'),
                                    glob('/lib/libz.so*'),
-                                   glob('/lib64/libz.so*')  ); @f > 1 ? 1 : 0 },
+                                   glob('/lib64/libz.so*'),
+                                   glob('/lib/i386-linux-gnu/libz.so*'),
+                                   glob('/usr/lib/i386-linux-gnu/libz.so*'),
+                                   glob('/usr/lib32/libz.so*')  ); @f > 1 ? 1 : 0 },
     'zlib-devel' => sub { -f '/usr/include/zlib.h' },
 };
 
